include/tlog/errs.h

```diff
@@ -141,4 +141,68 @@ extern void tlog_errs_pushf(struct tlog_errs **perrs, const char *fmt, ...)
  */
 extern int tlog_errs_print(FILE *stream, const struct tlog_errs *errs);
 
+/**
+ * Push an error to the perrs stack, with message as a constant string
```

Comment:
Since we're referring to the literal name "perrs" here, it would be better to quote it with double quotes.